third_party/googletest/src/googletest/src/gtest.cc

```diff
@@ -3978,6 +4117,19 @@ class XmlUnitTestResultPrinter : public EmptyTestEventListener {
   GTEST_DISALLOW_COPY_AND_ASSIGN_(XmlUnitTestResultPrinter);
 };
 
+#if GTEST_OS_STARBOARD
+void WriteOuputFile(const std::string &output_file, const std::string &data) {
+  SbFileError err;
+  starboard::ScopedFile cache_file(
+      output_file.c_str(), kSbFileCreateAlways | kSbFileWrite, NULL, &err);
+  // TODO: Change to SB_DCHECK once all platforms are verified
```

Comment:
Is this an old TODO ? Please change it to a TODO(b/1234) format so there's a bug to follow up on it -if it's at all relevant anymore